pyproject.toml

```diff
@@ -54,7 +54,7 @@ dependencies = [
     "pyhanko-certvalidator>=0.19.5",
     "pypdf>=3.1.0",
     "python-bidi>=0.4.2",
-    "reportlab>=4.0.4,<4.1",
+    "reportlab>=4.0.4",
```

Comment:
Tt might be wise to pin `reportlab` to `<5` (or perhaps even `<4.2`), so that things like #751 are less likely to happen again in the future.

Reply:
I thought about this, but have not been sure as I do not know how strictly *reportlab* follows semantic versioning and to which extent.

Reply:
If they tend to make breaking things in patch releases, I don't see why they wouldn't do so in minor or major releases. With this in mind, you could even consider an exact version restriction, just to be safe.

Reply:
Please avoid strict pinning in a library.
As a project-developer , you can always use your own strictrer pinning, but you cannot do the opposite.

Reply:
@woutervh For unnecessarily strict dependency restrictions I agree. 
But in this case, I'd rather have reportlab pinned to `<5` - I'd rather have an outdated sub-dependency, than a broken direct dependency that crashes my application. 
I don't think it's reasonable to expect anyone to have to pin each and every one of their dependency's dependencies. 
In the worst case, you can always fork the repo and change the restriction yourself. I have done this many times myself, actually.

Reply:
I have added another commit which pins the major release at least. Given the current maintenance state of *xhtml2pdf* and having to regularly deal with too strict version pinning myself, this should be a suitable trade-off for now.